contracts/PetPark.sol

```diff
@@ -4,4 +4,59 @@ pragma solidity ^0.8.0;
 
 contract PetPark {
 
-}
\ No newline at end of file
+    address owner;
+
+    enum Gender {Male, Female}
+    enum AnimalType {None, Fish, Cat, Dog, Rabbit, Parrot}
+
+    struct Person {
+        uint Age;
+        Gender gender;
+        }
+
+    mapping (uint => uint) CountByAnimalType;
+    mapping (address => Person) PersonByAddress;
+    mapping (address => uint) BorrowedByAddress;
+
+    event Added (uint indexed animalType, uint indexed count);
+    event Borrowed (uint indexed animalType);
+    event Returned  (uint indexed animalType);
+
+    constructor() {
+        owner = msg.sender;
+    }
+    
+    modifier onlyOwner() {
+        require(msg.sender == owner, "No animals owned.");
```

Comment:
The revert string here should be - `Not an owner`

Reply:
oh sorry I missed that part of the instruction - I have adjusted all the revert strings